src/utils.ts

```diff
@@ -24,5 +24,5 @@ export const join = (names: Array<string | undefined>) => {
 };
 
 export const isPopulated = (value: any): boolean => {
-  return typeof value !== undefined && value !== null && value !== "";
+  return value !== undefined && value !== null && value !== "";
```

Comment:
Is this change necessary?

Reply:
The fields were always coming up populated, with no value.

Reply:
Gotcha, I now realize that `undefined` was supposed to be quoted, nice catch.